rocketpool-daemon/node/node.go

```diff
@@ -171,13 +173,15 @@ func (t *TaskLoop) Run() error {
 }
 
 // Stop the daemon
-func (t *TaskLoop) Stop() {
+func (t *TaskLoop) Stop() error {
 	if t.metricsServer != nil {
 		// Shut down the metrics server
 		ctx, cancel := context.WithTimeout(context.Background(), metricsShutdownTimeout)
 		defer cancel()
-		t.metricsServer.Shutdown(ctx)
+		return t.metricsServer.Shutdown(ctx)
```

Comment:
This was "intentionally" left this way because we're shutting the server down anyway so if the HTTP server errors out during shutdown there's nothing to be done about it but it doesn't hurt to log it. Let's add a wrapper noting the error happened during HTTP server shutdown here.